src/main/scala/mesosphere/marathon/core/group/GroupManagerModule.scala

```diff
@@ -17,7 +19,7 @@ import scala.concurrent.ExecutionContext
 class GroupManagerModule(
     config: GroupManagerConfig,
     scheduler: Provider[DeploymentService],
-    groupRepo: GroupRepository)(implicit ctx: ExecutionContext, eventStream: EventStream) {
+    groupRepo: GroupRepository)(implicit ctx: ExecutionContext, eventStream: EventStream, authorizer: Authorizer) {
```

Comment:
Where is the authorizer used?

Reply:
GroupApiService

Reply:
Does it make sense to make `authorizer` a dependency of the `GroupApiService` class then?

Reply:
it is dependency of GroupApiService, but in order to inject that dependency, I need it to be available in the GroupManagerModule